README.md

```diff
@@ -118,10 +118,10 @@ All probes are disabled when using sampling or memory allocation profiling.
 
 ### Java Flight Recorder
 
-JFR provides low overhead CPU, allocation, IO wait and lock profiling and runs on all major operating systems. It is available on Oracle JDK since Java 7 and on OpenJDK since Java 11. 
+JFR provides low overhead CPU, allocation, IO wait and lock profiling and runs on all major operating systems. It is available on Oracle JDK since Java 7 and on OpenJDK since Java 11 (make sure you have at least [11.0.3](https://bugs.openjdk.java.net/browse/JDK-8219347)).
+
 To our knowledge, it is the only low-overhead allocation profiler for Windows.
 However, it has several shortcomings: It will not sample native method calls, so you will get misleading CPU results if your code does a lot of system calls (like reading files). 
```

Comment:
This sentence isn't correct anymore, we just know one shortcoming now :) Can you overhaul that section too?

Reply:
so by several you mean you'll list them all, ok, lemme' fix it